packages/ketcher-core/src/application/editor/data/monomers.ket

```diff
@@ -44199,6 +44199,9 @@
     "id": "SS3___Dipropanol-disulfide",
     "fullName": "Dipropanol-disulfide",
     "alias": "SS3",
+     "idtAliases": {
+    "base": "3ThioMC3-D"
+  },
```

Comment:
1) This monomer has only third position alias. Please set this object:
```
"idtAliases": {
      "base": "ThioMC3-D",
      "modifications": {
        "endpoint3": "3ThioMC3-D",
      }
    },
```

2) Please correct formatting (I mean tabs)